src/options/use-class-literals.ts

```diff
@@ -0,0 +1,12 @@
+import { BooleanSupportOption } from 'prettier';
+import { CATEGORY_PUG } from '.';
+
+export const PUG_USE_CLASS_LITERALS: BooleanSupportOption = {
+	since: '1.10.0',
+	category: CATEGORY_PUG,
+	type: 'boolean',
```

Comment:
As discussed in the issue, we should rewrite this to a `choice` (same for _id literal_)

Reply:
I was just doing that 😂